core/controllers/resources_test.py

```diff
@@ -914,26 +913,6 @@ def test_release_coordinator_able_to_update_promo_bar_config(self) -> None:
 
         self.logout()
 
-    def test_update_promo_bar_rules_with_unexpected_exception_returns_error(
```

Comment:
Still should be fine to keep this test, right? Though I don't know what you mean by "unexpected exception" here -- try to explain a bit, in a comment, what's wrong with the payload.

Reply:
No harm keeping it, but I'm ok dropping it too. Your call.